microcosm_pubsub/main.py

```diff
@@ -72,3 +80,39 @@ def load_config(metadata):
     for message in messages:
         with message:
             print message.content  # noqa
+
+
+class SimpleConsumerDaemon(ConsumerDaemon):
```

Comment:
This is basically the example from the README with some timing information printed on consume.